MOBILE_API.xml

```diff
@@ -2479,6 +2480,22 @@
       <param name="state" type="WindowState" mandatory="true"/>     
     </struct>
 
+    <struct name="SeatStatus" since="7.1">
+      <description>Describes the status of a parameter of seat.</description>
+      <param name="seatLocation" type="SeatLocation"  mandatory="true"/>
+      <param name="conditionActive" type="Boolean"  mandatory="true"/>
+    </struct>
+
+    <struct name="SeatOccupancy" since="7.1">
+      <param name="seatsOccupied" type="SeatStatus" array="true" minsize="0" maxsize="100" mandatory="false">
+        <description>Seat status array containing location and whether the seats are occupied.</description>
+      </param>
+      <param name="seatsBelted" type="SeatStatus" array="true" minsize="0" maxsize="100" mandatory="false">
+        <description>Seat status array containing location and whether the seats are belted.</description>
+      </param>
+    </struct>
+
```

Comment:
@ychernysheva please remove extra blank line

Reply:
@AKalinich-Luxoft Deleted in fc601bd